src/v/net/conn_quota.cc

```diff
@@ -25,8 +25,10 @@ conn_quota::units::~units() noexcept {
     }
 }
 
-conn_quota::conn_quota(conn_quota::config_fn cfg_f) noexcept
-  : _cfg(cfg_f()) {
+conn_quota::conn_quota(
+  conn_quota::config_fn cfg_f, seastar::logger* log) noexcept
```

Comment:
nit: here and elsewhere, use `ss::` namespace (`ss::logger`) instead of `seastar::`.

Reply:
fixed